Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,39 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main(string[] args) //First Commit
         {
-            Console.WriteLine("Hello World!");
+       
+            do
+            {
+                string firstname;
+                Console.WriteLine("What is your Firstname?   ");
+                firstname = Console.ReadLine();
```

Comment:
You can declare and assign variables at once.
Like that: `string firstname = Console.ReadLine();`

It would be more readable.